packages/base/src/3dview/mainview.tsx

```diff
@@ -614,11 +614,17 @@ export class MainView extends React.Component<IProps, IStates> {
         ) {
           continue;
         }
-        const intersectMesh = intersect.object.name.includes('-front')
-          ? intersect.object.parent.getObjectByName(
-              intersect.object.name.replace('-front', '')
-            )
-          : intersect.object;
+        let intersectMesh = intersect.object;
+        if (intersect.object.name.includes('-front')) {
```

Comment:
Relying on the name is a bit of a poor choice. We should have UUIDs for object names on the long term to prevent any issue.